### ISSUE TITLE
Generate_jwt is attempting to decode the token as utf-8

### ISSUE THREAD

**Issue by bmustill-rose** [prschmid/zoomus#157]
Using zoomus 1.1.3 installed via pip:

```
Python 3.8.6 (tags/v3.8.6:db45529, Sep 23 2020, 15:37:30) [MSC v.1927 32 bit (Intel)] on win32
Type "help", "copyright", "credits" or "license" for more information.
>>> import json
>>> from zoomus import ZoomClient
>>> client = ZoomClient('xxx', 'yyy')
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "C:\Users\Ben\AppData\Local\Programs\Python\Python38-32\lib\site-packages\zoomus\client.py", line 62, in __init__
    "token": util.generate_jwt(api_key, api_secret),
  File "C:\Users\Ben\AppData\Local\Programs\Python\Python38-32\lib\site-packages\zoomus\util.py", line 262, in generate_jwt
    return token.decode("utf-8")
AttributeError: 'str' object has no attribute 'decode'
```

As per the above, `generate_jwt` calls `jwt.encode` to generate the token which returns an str. However, `generate_jwt` then attempts to decode `token` as utf-8 when it returns it which is no longer required / supported which in turn causes the above error.

Modifying `generate_jwt` to return `token` without attempting to decode it works for me. I'm reluctant to submit a PR though as I only discovered this project a few hours ago so am not really in a position to say conclusively if this is the right solution or not.

**Comment by mch1024** [comment]
Same with `Python 3.6.9 (default, Oct  8 2020, 12:12:24) [GCC 8.4.0] on linux`, v 1.1.3.

**Comment by ruiribeiro04** [comment]
Same on `Python 3.7.9 (tags/v3.7.9:13c94747c7, Aug 17 2020, 18:58:18) [MSC v.1900 64 bit (AMD64)] on win32`

**Comment by GalNakash-RecoLabs** [comment]
Same on Python 3.8.
You are trying to decode an object that is already decoded.
You have str, there is no need to decode from UTF-8 anymore.

**Comment by pablolgit** [comment]
Same on `Python 3.9.0 (default, Nov 18 2020, 13:28:38) [GCC 8.3.0]`
I've modified `util.py` to only return `token` and that worked for my use case, not sure if it would break anything else.

**Comment by mch1024** [comment]
The builds are also failing because of this https://travis-ci.org/github/prschmid/zoomus/jobs/751150219

It could break Python 2 code but I don't think Python 2 should be the target.

**Comment by TheTechAlchemist** [comment]
I confirm this problem on Python 3.6.7 with PyJWT 2.0.1 on GNU/Linux; it works without errors with pyjwt 1.7.1

**Comment by cchacholiades** [comment]
Same issue here, how can we get around this? 

Platform: **Python 3.6 running on 64bit Amazon Linux/2.9.17**

**Comment by prschmid** [comment]
Hello everyone! Terribly sorry about the lack of communication on my part! I've been super busy (work, family, etc) recently and haven't had the time to work on this. I am in the process of updating the dependencies right now and will remove the `decode()` part form `util.py` which is causing the issue. Stay tuned and we should have this resolved shortly.

**Comment by prschmid** [comment]
Waiting for [Travis to run the tests](https://travis-ci.org/github/prschmid/zoomus/jobs/756731517), but it looks like they are having a bit of a backlog and so things are queued up.

![Screen Shot 2021-01-29 at 11 28 49 AM](https://user-images.githubusercontent.com/710710/106301072-407a4900-6225-11eb-8e84-b0db265e4ce2.png)

**Comment by prschmid** [comment]
This has been fixed in bd79bef4474db7486b36138248876dcc3741683b and the newest `zoomus` lib that should be pushed to pypi should not have this issue anymore. Thank you for everyone's patience!

**Comment by patcon** [comment]
Thanks so much @prschmid! I'm glad you're getting family time, and I'm really grateful for the time you found for this after the important stuff :)
